### PR TITLE
Remove the step of CSV to Parquet

### PR DESCRIPTION
**Pull request by GaryShen2008**
Rename the variable trans_data to test_data
Add spark.conf to support reading CSV on GPU
Remove some XGBoost parameters, use default value

Signed-off-by: gashen <gashen@nvidia.com>

### CODE REVIEW

File: examples/Spark-ETL+XGBoost/mortgage/notebooks/python/MortgageETL+XGBoost.ipynb [hunk 1]
```diff
@@ -738,7 +668,6 @@
     "    splits[1].write.parquet(output_path_test, mode=\"overwrite\")\n",
     "\n",
     "# print explain\n",
```
Comment (wbo4958) [inline examples/Spark-ETL+XGBoost/mortgage/notebooks/python/MortgageETL+XGBoost.ipynb:670]: Seems this comment is not needed again.

File: examples/Spark-ETL+XGBoost/mortgage/notebooks/python/MortgageETL+XGBoost.ipynb [hunk 2]
```diff
@@ -738,7 +668,6 @@
...
-    "# print(out.explain())\n",
     "end = time.time()\n",
     "print(\"ETL takes {}\".format(end - start))"
```
Comment (wbo4958) [inline examples/Spark-ETL+XGBoost/mortgage/notebooks/python/MortgageETL+XGBoost.ipynb:672]: could we move the time metric into `if is_save_dataset:`

**Review by wbo4958 — COMMENTED** [review]
LGTM

File: examples/Spark-ETL+XGBoost/mortgage/notebooks/python/MortgageETL+XGBoost.ipynb (hunk 1 above)
Comment (GaryShen2008) [reply examples/Spark-ETL+XGBoost/mortgage/notebooks/python/MortgageETL+XGBoost.ipynb]: removed

File: examples/Spark-ETL+XGBoost/mortgage/notebooks/python/MortgageETL+XGBoost.ipynb (hunk 2 above)
Comment (GaryShen2008) [reply examples/Spark-ETL+XGBoost/mortgage/notebooks/python/MortgageETL+XGBoost.ipynb]: Move into if section.

**Review by wbo4958 — DISMISSED** [review]

**Comment by nvliyuan** [comment]
Do we need to reimport all libraries in `XGBoost Spark with GPU` part? It seems already imported some libraries at first.

**Comment by GaryShen2008** [comment]
> Do we need to reimport all libraries in `XGBoost Spark with GPU` part? It seems already imported some libraries at first.

Good catch. Removed duplicated import.

**Review by nvliyuan — APPROVED** [review]

**Pull request merged**